src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,36 @@
-import React from 'react';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+import { Post } from '../../types/Post';
+
+import './PostInfo.scss';
+
+type Props = {
```

Comment:
It is matter of my preference, but I always give meaningful names based on components name to props interfaces e.g.: `PostInfoProps`, especially if for some reason you would have to export that interface